app/routers/event.py

```diff
@@ -84,6 +90,60 @@ async def eventedit(request: Request) -> Response:
     return templates.TemplateResponse("eventedit.html", {"request": request})
 
 
+@router.get("/edit/view_countries", include_in_schema=False)
+async def choose_country(request: Request,
+                         session=Depends(get_db)) -> Response:
+    """
+    Displays the list of all countries name from "Country" table.
+    """
+    countries_names = get_all_countries_names(session=session)
+    return templates.TemplateResponse("eventedit.html",
+                                      {"request": request,
+                                       "countries_names": countries_names})
+
+
+@router.post("/edit/view_countries", include_in_schema=False)
+async def check_country_time(request: Request,
+                             session=Depends(get_db)) -> Response:
+    """
+    Displays datalist of all countries name from "Country" table.
+    By clicking "Check time" the converted meeting time is displayed.
+    If the user's ip is not recognized, an error message will appear.
+    ** temporarily using a random israeli ip address instead **
+    """
+    # TODO: define ip as request.client.host
+    # ip = request.client.host
+    random_israeli_ip_for_now = '82.166.236.10'
+    ip = random_israeli_ip_for_now
+    match = geolite2.lookup(ip)
+    if match is not None:
+        data = await request.form()
+        country = data['countries']
+        start_time = dt.strptime(
+            data['start_date'] + ' ' + data['start_time'],
+            TIME_FORMAT)
+        end_time = dt.strptime(
+            data['end_date'] + ' ' + data['end_time'],
+            TIME_FORMAT)
+        user_timezone = match.timezone
+        meeting_time_for_invitee = get_meeting_local_duration(
+                                    start_time=start_time,
+                                    end_time=end_time,
+                                    user_timezone=user_timezone,
+                                    country=country,
+                                    session=session)
```

Comment:
Just `start_time, end_time, user_timezone, country, session`. The keywords are redundant in this case.